pkg/metrics/metrics.go

```diff
@@ -5,6 +5,12 @@ import (
 	"k8s.io/component-base/metrics/legacyregistry"
 )
 
+// The various stability levels used here are analogous to the ones used in the Kubernetes project. They are:
+// INTERNAL metrics have no stability guarantees, as such, labels may be arbitrarily added/removed and the metric may be deleted at any time.
+// ALPHA metrics have no stability guarantees, as such, labels may be arbitrarily added/removed, while metrics undergo a deprecation cycle before deletion.
+// BETA metrics have some stability guarantees, as such, labels may be added but not removed, while metrics undergo a deprecation cycle before deletion.
+// STABLE metrics are guaranteed not be mutated.
+
```

Comment:
Not sure we want to promise any deprecation cycle or stability guaranty. We typically give no stability guaranty for metrics or labels.

Reply:
ACK, in that case, do you think we can go with [`metrics.INTERNAL`](https://github.com/openshift/cluster-monitoring-operator/pull/2153/files#diff-4bc91a74bc47b8467cdcd25a7e3fe1651dbe44907f478377eca31b982f444f23R9) for all meta metrics defined here? I say this because of two reasons.

* `ALPHA` metrics undergo a deprecation cycle, unlike `INTERNAL` ones.
* `ALPHA` is the [default](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/component-base/metrics/opts.go#L92) stability level.

Owing to these we could move `BETA` and `ALPHA` meta metrics to an `INTERNAL` stability level (no deprecation cycles and no stability guarantees), while being compliant with `component-base`.

Reply:
(bump)

Reply:
(ping) Requesting inputs on this.

Reply:
Sure I'm open to that, it makes sense. But I'd suggest we don't solve this in this PR here. This can probably wait for the 4.17 dev cycle. Until then we can just go with ALPHA like we have in the past?

Reply:
Sure, I've pushed the changes.